accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/history/InstallationLogger.java

```diff
@@ -11,10 +11,6 @@
 
     void addVerboseMessage(Logger log, String message);
 
-    void addError(final String error);
-
-    void addError(Logger log, String error);
```

Comment:
@kwin Shouldn't we leave that one in for consistency? It should log to log file and execution log (in the same way the other log severities allow for this)

Reply:
This is used nowhere in the code!